Kentico.Kontent.Delivery.Abstractions/SharedModels/ISystemAttributes.cs

```diff
@@ -3,28 +3,13 @@
 namespace Kentico.Kontent.Delivery.Abstractions
 {
     /// <summary>
-    /// Represents system attributes of any object in Kentico Kontent.
+    /// Represents system attributes of ContentItem, ContentType and Taxonomy objects in Kentico Kontent.
     /// </summary>
-    public interface ISystemAttributes
+    public interface ISystemAttributes : ISystemBaseAttributes
     {
-        /// <summary>
-        /// Gets the codename of the object.
-        /// </summary>
-        string Codename { get; }
-
-        /// <summary>
-        /// Gets the identifier of the object.
-        /// </summary>
-        string Id { get; }
-
         /// <summary>
         /// Gets the time the object was last modified.
         /// </summary>
         DateTime LastModified { get; }
```

Comment:
after an internal discussion:
languages on their own don't contain the "last modified" timestamp. the last modified time is store per the whole language settings object so it's global for all languages in a given project. rather than introducing inconsistency in the behavior of a last_modified property, we'll go with an inconsistency in the presence of such a field.